libs/plutus-preprocessor/plutus-preprocessor.cabal

```
@@ -43,7 +43,7 @@ library
         plutus-ledger-api,
         template-haskell
 
-    if (impl(ghc <9.6) || impl(ghc >=9.7))
+    if (impl(ghc <9.6) || impl(ghc >=9.7) || os(windows))
```

Seems like according to cabal formatter, it should be like this: 
```suggestion
    if ((impl(ghc <9.6) || impl(ghc >=9.7)) || os(windows))
```